ambari-server/src/main/java/org/apache/ambari/server/controller/KerberosHelperImpl.java

```diff
@@ -3913,6 +3935,11 @@ public long createStages(Cluster cluster,
       addUpdateConfigurationsStage(cluster, clusterHostInfoJson, hostParamsJson, event, commandParameters,
         roleCommandOrder, requestStageContainer);
 
+      // *****************************************************************
+      // Create stage to trigger a topology update event being published
+      addTopologyUpdateTriggerStage(cluster, clusterHostInfoJson, hostParamsJson, event, commandParameters,
+        roleCommandOrder, requestStageContainer);
```

Comment:
`KerberosHelperTest` needs to be changed accordingly.

Reply:
ok

Reply:
Done